### PR TITLE
New shutdown script for proper power-off

### PR DESCRIPTION
**Pull request by schmurtzm**
- New `shutdown.sh` script which runs automatically totally detached from default shell.
- No more FAT "dirty flag" when power off.
- Kill all background tasks with hook to `/mnt/SDCARD` (SIGTERM for proper close then SIGKILL)
- Unmount all necessary volumes / files related to `/mnt/SDCARD`
- The argument "`-r`" allows to reboot instead of shutdown

**Bonus:**
- Changed "power cutoff time" to 10 secs for MMP
- Simplified hold action on POWER button
- Disabled SELECT+L2/R2 brightness shortcut for MMP (also has MENU+VOL for brightness)

### CODE REVIEW

**Review by Aemiii91 — DISMISSED** [review]

**Comment by Aemiii91** [comment]
Should we get the MainUI binaries from @tGecko merged into this first? And maybe implement this: https://discord.com/channels/529983248114122762/1007796369785897090/1123387351389982750

**Comment by schmurtzm** [comment]
> Should we get the MainUI binaries from @tGecko merged into this first? And maybe implement this: [discord.com/channels/529983248114122762/1007796369785897090/1123387351389982750](https://discord.com/channels/529983248114122762/1007796369785897090/1123387351389982750)

I suggest to rename the script "`shutdown`" and put it in `bin` folder. Then @tGecko could replace `/customer/app/axp_test poweroff`  in MainUI binaries just with `shutdown`.

File: src/keymon/keymon.c [hunk 1]
```diff
@@ -174,13 +174,6 @@ void shutdown(void)
     system_clock_save();
     sync();
 
-    if (DEVICE_ID == MIYOO283) {
-        reboot(RB_AUTOBOOT);
-    }
-    else if (DEVICE_ID == MIYOO354) {
-        system("poweroff");
-    }
-
```
Comment (Aemiii91) [inline src/keymon/keymon.c:183]: We need to put `system("shutdown")` here as well.

**Review by Aemiii91 — CHANGES_REQUESTED** [review]

File: src/keymon/keymon.c (hunk 1 above)
Comment (schmurtzm) [reply src/keymon/keymon.c:183]: The flag is not enough ? 
https://github.com/OnionUI/Onion/blob/6d1cb0d7f111ca360aaf3e91611a9237adb42bd2/src/common/system/system.h#L29
It already shutdown like that thanks to runtime.sh 🤔

Comment (Aemiii91) [reply src/keymon/keymon.c:183]: No, this function is used to force shutdown (when runtime.sh is locked by a process not terminating)

**Review by Aemiii91 — APPROVED** [review]

**Pull request merged**